src/components/Navbar.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PageNavLink } from './PageNavLink';
+
+export const Navbar: React.FC = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <PageNavLink to="/" text="Home" />
+
+        <PageNavLink to="people" text="People" />
```

Comment:
The 'to' prop should start with a forward slash for the 'People' PageNavLink, so it should be '/people' instead of 'people'.